components/transactions/TransactionsDownloadCSV.js

```diff
@@ -44,11 +44,14 @@ const transformResultInCSV = json => {
     `Net Amount (${collectiveCurrency})`,
     'Subscription Interval',
     'Order Date',
+    'Tags',
   ].join(',');
 
   const lines = json.map(i => {
     const profile = `http://opencollective.com/${i.fromCollective.slug}`;
     const subscriptionInterval = i.subscription ? i.subscription.interval : 'one time';
+    const expenseTags = (i.expense && i.expense.category) || '';
```

Comment:
There is one small issue if we have an expense with multiple tags, only the first tag is returned in `expense.category`.